### PR TITLE
New version: Remeha.ServiceTool version 7.3.18.0

### PR DESCRIPTION
**Pull request by TutkuSilaVuralEyovgeAlten**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140359)

### CODE REVIEW

File: manifests/r/Remeha/ServiceTool/7.3.18.0/Remeha.ServiceTool.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    MinimumVersion: 3.1.32
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+    MinimumVersion: 14.38.33130.0
+ProductCode: '{08ED46A0-ABC1-461D-8044-339A3FF3FDF6}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://pcstwinget.blob.core.windows.net/pc-servicetool-images/RecomAppWix_7_3_18_0.msi
+  InstallerSha256: 31FD7395AF5D61F954C332BC3BB4F6AB79D8E19F8CC2F0F69094B62C478E42F1
+ManifestType: installer
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/r/Remeha/ServiceTool/7.3.18.0/Remeha.ServiceTool.installer.yaml:21]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/r/Remeha/ServiceTool/7.3.18.0/Remeha.ServiceTool.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Remeha.ServiceTool
+PackageVersion: 7.3.18.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/r/Remeha/ServiceTool/7.3.18.0/Remeha.ServiceTool.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/r/Remeha/ServiceTool/7.3.18.0/Remeha.ServiceTool.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Publisher: Remeha BV
+PublisherUrl: https://www.remeha.nl/
+Author: Remeha BV
+PackageName: Recom (PCST)
+License: MIT License
+ShortDescription: PC Servicetool allows for the inspection and management of Remeha boilers and heatpumps as well as long term monitoring of data.
+Description: Recom is a software program developed by Remeha B.V. It offers the possibility to communicate directly to Remeha boilers/appliances. The software shows temperatures, fan speed and the status of on/off components like pump and gas-air unit in a clear overview. In Recom , all parameters of the safety unit of the boiler are shown in one overview. Of course you can save settings on a hard drive, so you can always reset to default settings or use a certain setting for multiple boilers in a project.
+Moniker: recom
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/r/Remeha/ServiceTool/7.3.18.0/Remeha.ServiceTool.locale.en-US.yaml:16]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @TutkuSilaVuralEyovgeAlten,

This manifest version is depreciated. But good news - I've tested increasing the manifest version with no other changes, and the application still installs and launches normally. 

image

**Comment by TutkuSilaVuralEyovgeAlten** [comment]
> t version is depreciated. But good news - I've tested increasing the manifest version with no other c

Hello, I forgot to push one of my changes. We can ignore this version, I'll create a new one with the lates changes. Manifest version needs to be 1.6.0, right?

**Comment by stephengillie** [comment]
Sounds good, I'll close this PR. Yes, please use manifest version 1.6.0.

**Comment by stephengillie** [comment]
Close with reason: PR author will create a new PR for this.;

**Pull request closed without merging**